### ISSUE TITLE
MaterialUpdateListener error

### ISSUE THREAD

**Issue by alancomeau** [gocd/gocd#7953]
##### Issue Type
Bug Report

- Security issue? [**DO NOT** report it here. Use [hackerone](https://hackerone.com/gocd) or send a mail to support@thoughtworks.com]
- Bug Report
- Feature proposal
- Feature enhancement
- Documentation Report
- Question? [**DO NOT** ask it here. Ask in the [mailing list](https://groups.google.com/forum/#!forum/go-cd)].

##### Summary

snakeyaml parser error on commit message.
commit message below
--start commit message
ACNEW allow editing and updating the user and admin permissions
split the saving/updating of admin permissions into two separate methods and required Super User for the admin permissions update
added billing suspended as a new tab
 ----end commit message
ACNEW is the jira project name
##### Environment
ubuntu 18

###### Basic environment details

* Go Version: `Available in the footer`
GoCD Version: 20.1.0 (11114-f2c1771e333ea92dae82cd78d2efa1310c1086f0).

* JAVA Version: `Available from http://<go-server-host>/go/about`
JVM version: | 13.0.1

* OS: `Available from http://<go-server-host>/go/about`

OS Information: | Linux 4.4.0-173-generic

* Browser vendor and version (if relevant): `Hopefully not Internet Explorer 7 :)`
not relevant
###### Additional Environment Details

stacktrace from go-server.log
```
2020-04-01 17:12:44,663 WARN  [129@MessageListener for MaterialUpdateListener] MaterialDatabaseUpdater:127 - [Material Update] Modification check failed for material: URL: s
sh://git@stash.guild.com:7999/on/guild_restservice.git, Branch: master
org.yaml.snakeyaml.parser.ParserException: while parsing a block mapping
 in 'string', line 1959, column 3:
      commitHash: b06a58f06b6eedbcafc2 ... 
      ^
expected <block end>, but found Scalar
 in 'string', line 1967, column 5:
        split the saving/updating of adm ... 
        ^

        at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:571)
        at org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:157)
        at org.yaml.snakeyaml.parser.ParserImpl.checkEvent(ParserImpl.java:147)
        at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:227)
        at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:154)
        at org.yaml.snakeyaml.composer.Composer.composeSequenceNode(Composer.java:201)
        at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:152)
        at org.yaml.snakeyaml.composer.Composer.composeDocument(Composer.java:122)
        at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:105)
        at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:140)
        at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:524)
        at org.yaml.snakeyaml.Yaml.load(Yaml.java:437)
        at com.thoughtworks.go.domain.materials.git.GitModificationParser.parse(GitModificationParser.java:34)
        at com.thoughtworks.go.domain.materials.git.GitCommand.gitLog(GitCommand.java:133)
        at com.thoughtworks.go.domain.materials.git.GitCommand.modificationsSince(GitCommand.java:116)
        at com.thoughtworks.go.config.materials.git.GitMaterial.modificationsSince(GitMaterial.java:135)
        at com.thoughtworks.go.server.service.materials.GitPoller.modificationsSince(GitPoller.java:35)
        at com.thoughtworks.go.server.service.materials.GitPoller.modificationsSince(GitPoller.java:26)
        at com.thoughtworks.go.server.service.MaterialService.modificationsSince(MaterialService.java:128)
        at com.thoughtworks.go.server.materials.ScmMaterialUpdater.insertLatestOrNewModifications(ScmMaterialUpdater.java:56)
        at com.thoughtworks.go.server.materials.MaterialDatabaseUpdater.insertLatestOrNewModifications(MaterialDatabaseUpdater.java:157)
        at com.thoughtworks.go.server.materials.MaterialDatabaseUpdater.updateMaterialWithNewRevisions(MaterialDatabaseUpdater.java:149)
        at com.thoughtworks.go.server.materials.MaterialDatabaseUpdater$2.doInTransaction(MaterialDatabaseUpdater.java:108)
        at com.thoughtworks.go.server.transaction.TransactionCallback.doWithExceptionHandling(TransactionCallback.java:23)
        at com.thoughtworks.go.server.transaction.TransactionTemplate.lambda$executeWithExceptionHandling$2(TransactionTemplate.java:43)
        at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
        at com.thoughtworks.go.server.transaction.TransactionTemplate.executeWithExceptionHandling(TransactionTemplate.java:40)
        at com.thoughtworks.go.server.materials.MaterialDatabaseUpdater.updateMaterial(MaterialDatabaseUpdater.java:105)
        at com.thoughtworks.go.server.materials.MaterialUpdateListener.onMessage(MaterialUpdateListener.java:64)
        at com.thoughtworks.go.server.materials.MaterialUpdateListener.onMessage(MaterialUpdateListener.java:32)
        at com.thoughtworks.go.server.messaging.activemq.JMSMessageListenerAdapter.runImpl(JMSMessageListenerAdapter.java:83)
        at com.thoughtworks.go.server.messaging.activemq.JMSMessageListenerAdapter.run(JMSMessageListenerAdapter.java:63)
        at java.base/java.lang.Thread.run(Unknown Source)
```
##### Steps to Reproduce

1. 
2. 
3. 
4.

**Comment by maheshp** [comment]
There were similar issues reported earlier -https://github.com/gocd/gocd/issues/7752 
This issue has been fixed in GoCD 20.2.0. I will close this issue, if you face issues after upgrade please re-open with details.

**Comment by alancomeau** [comment]
that fixed our issue. Thank you
